src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.ts

```diff
@@ -12,6 +12,7 @@ import { ExerciseCategory } from 'app/entities/exercise-category.model';
 })
 export class HeaderExercisePageWithDetailsComponent implements OnInit, OnChanges {
     readonly IncludedInOverallScore = IncludedInOverallScore;
+    readonly moment = moment;
```

Comment:
Do you need this variable? I don't think you use it anymore.

Reply:
Yes, you are right. I forgot to delete it here. Done :-)